src/main/javascript/src/components/ListParticipants.vue

```diff
@@ -84,7 +85,23 @@ export default {
       this.tempSelectedInAGroup = [];
 
       this.moveToHandler(option, tempSelectedInAGroup);
+    },
+        handleOnChange(value) {
+      if (value === 0) {
+        this.tempSelectedInAGroup = this.listOfParticipants;
+      } else {
+        this.tempSelectedInAGroup = [];
+      }
     },
   },
 };
 </script>
+
+<style lang="scss">
```

Comment:
You should set this style tag to scoped like so:
`<style lang="scss" scoped>`

This will make it only apply to this component